### ISSUE TITLE
Error with Chromedriver assuming Chrome has crashed - running headless

### ISSUE THREAD

**Issue by berryt08** [T4rk1n/pytest-dash#64]
Getting the following error when I try to run pytest:

```
E       WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally
E         (unknown error: DevToolsActivePort file doesn't exist)
E         (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)
E         (Driver info: chromedriver=2.46.628388 (4a34a70827ac54148e092aafb70504c4ea7ae926),platform=Linux 4.9.0-7-amd64 x86_64)

venv/local/lib/python2.7/site-packages/selenium/webdriver/remote/errorhandler.py:242: WebDriverException
```

ChromeDriver 73.0.3683.20 
Google Chrome 72.0.3626.109 
chromedriver: /usr/bin/chromedriver /usr/local/bin/chromedriver
google-chrome: /usr/bin/google-chrome /usr/share/man/man1/google-chrome.1.gz

Wondering whether this has something to do with me running a headless server, e.g. the only way I can run google-chrome is with --headless

**Comment by T4rk1n** [comment]
> Wondering whether this has something to do with me running a headless server,

Maybe, there's no support for selenium options right now #58 so you can't set headless right now. 

Thinking of adding a `pytest_setup_selenium` hook, you would return a dict to give to the driver init.

conftest.py
```python
from selenium.webdriver.chrome.options import Options

def pytest_setup_selenium():
    options = Options()
    options.headless = True
    return {
        'executable_path': '/usr/bin/chromedriver',
        'chrome_options': options,
    }
```

**Comment by T4rk1n** [comment]
Released pytest-dash 2.1.0 with the hooks described above ^.

https://pytest-dash.readthedocs.io/en/latest/usage.html#pytest-setup-selenium

**Comment by berryt08** [comment]
@T4rk1n, thanks for the quick update!

A couple of points to note:
    1. As I am running Python 2.7, I had to edit in `application_runners.py` `class BaseDashRunner()` to `class BaseDashRunner(object)` to stop it failing when super() is called due to error `TypeError “argument 1 must be type, not classobj”` (https://stackoverflow.com/questions/1713038/super-fails-with-error-typeerror-argument-1-must-be-type-not-classobj-when)

    2. In your dash_threaded example in the docs you use PreventUpdate without importing it: `from dash.exceptions import PreventUpdate`

**Comment by T4rk1n** [comment]
@berryt08 Thanks for all the feedback, I applied the fix, tested on python 2 & released version 2.1.1
